### PR TITLE
docs: remove unnecessary step in tutorial

### LINKED ISSUE #25025: Tutorial Step Slightly Incorrect
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  
[ ] Performance issue
[ ] Feature request
[x] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead see https://github.com/angular/angular/blob/master/CONTRIBUTING.md#question
[ ] Other... Please describe:
</code></pre>

## Current behavior

HTTP section of the tutorial has a sentence that is slightly incorrect. After making some changes it asks us to test our changes providing the steps to take, but one of these steps is unnecessary.

## Expected behavior

The unnecessary step should be removed or better explained.

## Minimal reproduction of the problem with instructions

1. Open https://angular.io/tutorial/toh-pt6#update-heroes
2. Scroll to the last paragraph of that section ('Update Heroes')
3. It says the following sentence: "Refresh the browser, change a hero name, save your change, and click the "go back" button."
4. However we do not need to click the "go back" button as this is already integrated as part of our code in the same section. Below is the code from the same section of the tutorial that already provides this "go back" functionality automatically:
save(): void {
   this.heroService.updateHero(this.hero)
     .subscribe(() => this.goBack());
 } 

## What is the motivation / use case for changing the behavior?

Just to have the tutorial perfect as it has been so far until that sentence.

## Environment

<pre><code>
Angular version: X.Y.Z

Yes

Browser:
- [ ] Chrome (desktop) version XX
- [ ] Chrome (Android) version XX
- [x] Chrome (iOS) version XX
- [ ] Firefox version XX
- [ ] Safari (desktop) version XX
- [ ] Safari (iOS) version XX
- [ ] IE version XX
- [ ] Edge version XX
* Although I assume it is the same in other browsers too. 

For Tooling issues:
- Node version: XX  
- Platform:  

Others:

</code></pre>

### PR DESCRIPTION
**Pull request by FrederikSchlemmer**
closes #25025 

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] angular.io application / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: #25025 

## What is the new behavior?

The unnecessary step of clicking the "go back" button has been deleted. To avoid any confusion about the following navigation to the previous view after saving the change the bahaviour is explained.

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: aio/content/tutorial/toh-pt6.md [hunk 1]
```diff
@@ -294,8 +294,7 @@ That header is in the `httpOptions` constant defined in the `HeroService`.
   region="http-options">
 </code-example>
 
-Refresh the browser, change a hero name, save your change,
-and click the "go back" button. 
+Refresh the browser, change a hero name and save your change. The following navigation to the previous view is implemented in the `save()` method defined in `HeroDetailComponent`.
```
Comment (brandonroberts) [inline aio/content/tutorial/toh-pt6.md]: Change to `Navigating to the previous view ...`. "The following" makes it seem like there is code below this text.

**Review by brandonroberts — CHANGES_REQUESTED** [review]
Squash commits and update commit message to `docs: remove unnecessary step in tutorial for HTTP`

**Comment by brandonroberts** [comment]
Thanks for the PR @FrederikSchlemmer! I left you some feedback.

File: aio/content/tutorial/toh-pt6.md (hunk 1 above)
Comment (FrederikSchlemmer) [reply aio/content/tutorial/toh-pt6.md]: Thanks for the feedback. I changed this wording.

**Comment by FrederikSchlemmer** [comment]
@brandonroberts I squashed the new commits and renamed it. 

I'm not sure why the update commit of my fork is integrated in the pull request. If I squash this too, the update is counted with the files changed, so that the number increases to 345.

What's the best way to proceed?

**Comment by brandonroberts** [comment]
The best way is to rebase on master in your fork

1. Checkout and pull down latest `master` from upstream (git checkout master) (git pull upstream master)
2. Rebase your branch on top of master with the updated commit message (git checkout TourOfHeroesCorrection) (git rebase master -i)
3. Push your branch back up (git push origin --force-with-lease TourOfHeroesCorrection)

**Comment by FrederikSchlemmer** [comment]
@brandonroberts Thanks for the instructions! Everything is squashed in one commit with the updated commit message.

**Review by brandonroberts — APPROVED** [review]

**Pull request closed without merging**